articles/azure/azs-how-resize-os-drive.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: How to resize an unmanaged disk - PowerShell | UKCloud Ltd
-description: Provides help for resizing an unmanaged disk on UKCloud for Microsoft Azure
+title: How to resize an OS drive | UKCloud Ltd
+description: Provides help for resizing an OS drive on UKCloud for Microsoft Azure
 services: azure-stack
 author: Bailey Lawson
 toc_rootlink: Users
 toc_sub1: How To
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: Resize an unmanaged disk - PowerShell
-toc_fullpath: Users/How To/azs-how-resize-unmanaged-disk.md
-toc_mdlink: azs-how-resize-unmanaged-disk.md
+toc_title: How to resize an OS drive
+toc_fullpath: Users/How To/azs-how-resize-os-drive.md
+toc_mdlink: azs-how-resize-os-drive.md
 ---
 
-# How to resize an unmanaged disk - PowerShell
+# How to resize an OS drive with PowerShell
```

Comment:
How to resize an OS drive using PowerShell

Reply:
Fixed